### PR TITLE
Fixes #1: SUSI AI theme created

### LINKED ISSUE #1: Create sus ai theme
Create susi ai theme by taking inspiration from http://chat.susi.ai/overview

### PR DESCRIPTION
**Pull request by poonai**
susi ai theme created

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from setuptools import setup
+import os.path
+import sys
```
Comment (pri22296) [inline setup.py]: where is os.path and sys used?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  	platforms = 'any',
+  	classifiers = [
+    	"Framework :: Sphinx :: Extension",
+    	"Framework :: Sphinx :: Theme",
+    	"Intended Audience :: Developers",
+    	"Operating System :: OS Independent",
+    	"Topic :: Documentation :: Sphinx",
+    	"Topic :: Software Development :: Documentation",
+  	],
+    long_description = open('README.rst').read(),
```
Comment (pri22296) [inline setup.py]: fix indentation

File: setup.py (hunk 1 above)
Comment (poonai) [reply setup.py]: CTRL + A, CTRL +C, CTRL+V from `sphinx_fossasia_theme`

Comment (poonai) [reply setup.py]: I'll remove now

File: setup.py (hunk 2 above)
Comment (poonai) [reply setup.py]: updated.

File: setup.py (hunk 1 above)
Comment (poonai) [reply setup.py]: updated.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        'sphinx.html_themes': [
+            'sphinx_susiai_theme = sphinx_susiai_theme'
+        ],
+        'sphinx_themes': [
+            'path = sphinx_susiai_theme:get_html_theme_path'
+        ],
+    },
+    install_requires = ['sphinx>=1.3'],
+    platforms = 'any',
+    classifiers = [
```
Comment (pri22296) [inline setup.py:23]: Use proper indentation.
It should preferably look like this.
```python
classifiers = [
    "Framework :: Sphinx :: Extension",
    "Framework :: Sphinx :: Theme",
    "Intended Audience :: Developers",
    "Operating System :: OS Independent",
    "Topic :: Documentation :: Sphinx",
    "Topic :: Software Development :: Documentation",
],
```

Comment (poonai) [reply setup.py:23]: updated.

**Review by pri22296 — APPROVED** [review]

**Pull request merged**